libr/cons/input.c

```diff
@@ -230,6 +230,13 @@ R_API int r_cons_arrow_to_hjkl(int ch) {
 	return ch;
 }
 
+static inline void flush_istream(FILE *in_stream) {
+	int ch;
+	while ((ch = fgetc(in_stream)) != '\n' && ch != EOF) {
```

Comment:
missing space before (

Reply:
also, i dont think we want to call fgetc to skip chars in fgets(). did you tried your patch? because it seems to be like we can just remove the fflush line

Reply:
ok

Reply:
I have built it locally, it seems that r2 interactive run ok.